src/http-gateways/trustless-gateway.md

```diff
@@ -8,28 +8,36 @@ editors:
 
 # Trustless Gateway Specification
 
-Trustless Gateway is a minimal _subset_ of :cite[path-gateway]
+Trustless Gateway is a _subset_ of :cite[path-gateway]
 that allows light IPFS clients to retrieve data behind a CID and verify its
 integrity without delegating any trust to the gateway itself.
 
 The minimal implementation means:
 
-- data is requested by CID, only supported path is `/ipfs/{cid}`
-- no path traversal or recursive resolution, no UnixFS/IPLD decoding server-side
 - response type is always fully verifiable: client can decide between a raw block or a CAR stream
+- no UnixFS/IPLD decoding server-side
+- for CAR files:
+  - the behavior is identical to :cite[path-gateway]
+- for raw blocks:
+  - data is requested by CID, only supported path is `/ipfs/{cid}`
+  - no path traversal or recursive resolution
```

Comment:
This seems confusing. It seems like the minimal implementation is just to support blocks and not CARs. CARs is going to support some IPLD or UnixFS decoding server side.

Reply:
Yes, this looks like a bad mental shortcut.
I've replaced  "no UnixFS/IPLD decoding server-side" with "no UnixFS/IPLD deserialization"